dynawo/3rdParty/CMakeLists.txt

```diff
@@ -904,12 +901,21 @@ else()
   set(LIBZIP_HOME ${install_dir})
 endif()
 
-# if ( CXX11_ENABLED )                                                                              # New part for IIDM-2
+#-wip- if ( CXX11_ENABLED )                                                                              # New part for IIDM-2
+
+# Minimum required for powsybl that sets that rule
+cmake_minimum_required(VERSION 3.12)
+
+include( "Support-LibXml2")
```

Comment:
when the correspondign file is moved it should be something like this include(libxml2/LibXml2.cmake)

Reply:
that may be discussed. Not all the 3rd parties have a directory in the CMAKE_SOURCE_DIR or whatever. All our "finders" like FindLibZIP.cmake are under the same directory 3rdParty/cmake. It's simplier to follow that same rule. Support-qqch.cmake is a "finder" that configures installation when it does not find, which is quite the same. Do you follow me ?

Reply:
For me your support is not the same as a FindXXX in the cmake folder so it does not belong in the same folder

Reply:
We may discuss upon that point. Using one single and existing directory makes it a lot easier, as it is the case for FindXXX. Easier since we don't create a new directory, easier for adding new SupportXXX since one may find it exactly in our "modules" directory. Existing libxxx directories under 3rdPartiy are only for patches. SupportXXX is very different from patches. In fact it is only a wrapper of FindXXX that generates target for installing when it does not find. I still thing it is similar, that's why I borrow the concept from the way cmake keeps its modules: choosing a unique name and putting all modules in one single directory.